backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserAccountCreationController.java

```diff
@@ -109,8 +114,10 @@ public void enrollVoiceCallMfa(
    */
   @PostMapping("/enroll-email-mfa")
   public void enrollEmailMfa(
-      @RequestBody EnrollMfaRequest requestBody, HttpServletRequest request) {
-    // WIP: doesn't interact with Okta yet.
+      @RequestBody EnrollMfaRequest requestBody, HttpServletRequest request) throws OktaAuthenticationFailureException {
+    String userId = getUserId(request.getSession());
+    String factorId = _oktaAuth.enrollEmailMfa(userId, requestBody.getUserInput());
+    request.getSession().setAttribute(FACTOR_ID, factorId);
```

Comment:
It might be worth combining all of these `/enroll-` endpoints and adding a type to `EnrollMfaRequest`.

Reply:
I do quite like that idea! I didn't realize when I first started this how similar all the endpoints would be. But I know @nickclyde has already set up the frontend to ping these endpoints and I'm not sure how difficult it would be to change - Nick, any thoughts on this?